transport/graphsync/graphsync.go

```diff
@@ -617,53 +502,37 @@ func (t *Transport) gsReqRecdHook(p peer.ID, request graphsync.RequestData, hook
 	}
 
 	// Check if the callback indicated that the channel should be paused
-	// immediately
+	// immediately (eg because data is still being unsealed)
 	paused := false
 	if err == datatransfer.ErrPause {
 		paused = true
+		log.Warnf("process new request: pause response")
```

Comment:
Can we add some identifying information about the request in this message? Maybe `chid`?